delivery_carrier_return_barcode_pattern/models/delivery_carrier.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import re
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+HELP_RETURN_BARCODE_PATTERN = """
```

Comment:
@mmequignon Would this be translated as is ?

Reply:
Yes, why not?

Reply:
That's what I said.

Here under, the commit that update the pot file:

https://github.com/OCA/delivery-carrier/commit/aaffa4de50fa8bf187538720eeee4b8412c8f859

@mmequignon You should call explicitly translation function for strings that are not directly defined on fields property but through a variable.

Could you do the fix ?